openapi/bitbucket/client.bal

```diff
@@ -331,3 +374,37 @@ public client class Client {
          _ = check self.clientEp-> delete(path, request, targetType =http:Response);
     }
 }
+
+# Generate query path with query parameter.
+#
+# + queryParam - Query parameter map
+# + return - Returns generated Path or error at failure of client initialization
+isolated function  getPathForQueryParam(map<anydata>   queryParam)  returns  string {
+    string[] param = [];
+    param[param.length()] = "?";
+    foreach  var [key, value] in  queryParam.entries() {
+        if  value  is  () {
+            _ = queryParam.remove(key);
+        } else {
+            if  string:startsWith( key, "'") {
+                 param[param.length()] = string:substring(key, 1, key.length());
+            } else {
+                param[param.length()] = key;
+            }
+            param[param.length()] = "=";
+            if  value  is  string {
+                string updateV =  checkpanic url:encode(value, "UTF-8");
```

Comment:
Is there a reason to use checkpanic instead of check?

Reply:
This is an auto-generated code, I'll try with the latest tool again

Reply:
Resolved manually